edb/pgsql/metaschema.py

```diff
@@ -5267,7 +5207,88 @@ def _generate_sql_information_schema() -> List[dbops.View]:
         SELECT pg_type.*
         FROM pg_type
         JOIN pg_namespace pn ON pg_type.typnamespace = pn.oid
-        WHERE nspname IN ('pg_catalog', 'pg_toast')
+        WHERE nspname IN ('pg_catalog', 'pg_toast', 'information_schema')
+        UNION ALL
+        SELECT pt.oid,
+            vt.table_name,
+            edgedbsql.uuid_to_oid(vt.module_id) as typnamespace,
+            typowner,
+            typlen,
+            typbyval,
+            typtype,
+            typcategory,
+            typispreferred,
+            typisdefined,
+            typdelim,
+            typrelid,
+            typsubscript,
```

Comment:
Looks like `typsubscript` is only added in [PostgreSQL 14](https://www.postgresql.org/docs/14/catalog-pg-type.html), thus this breaks [using EdgeDB on PostgreSQL 13](https://github.com/edgedb/edgedb/actions/runs/3992636910/jobs/6857300141), which will still have a (final) release on [Nov 2025](https://www.postgresql.org/support/versioning/). @elprans should we fix the use of `typsubscript` or drop the support of PostgreSQL 13?

Reply:
We can just remove `typsubscript` from here. If this is a change in recent pg version, no tool will depend on it.